### PR TITLE
[WIP] vs2017 taken from defaults

### PR DESCRIPTION
**Pull request by hmaarrfk**
closes https://github.com/conda-forge/staged-recipes/issues/4474

[circle skip] 

[travis skip]

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 skip_commits:
   message: /^Merge pull request /
 
+image:
+    - Visual Studio 2017
```
[the remark this answers is not in the document]
Comment (hmaarrfk) [reply .appveyor.yml]: @isuruf it seems that I actually need to make this change to build this recipe correctly on appveyor. Any ideas?

**Comment by jakirkham** [comment]
As for the runtime libraries, we have things like [this recipe]( https://github.com/conda-forge/vs2015_runtime-feedstock/tree/master/recipe ). We will also need to add `vs2017_runtime` to hold the newer libraries.

**Comment by hmaarrfk** [comment]
I think this one recipe creates the runtimes as well. 

Defaults just called it vs2015 as it is forward compatible I think

**Comment by msarahan** [comment]
It needs to be vs2015_runtime because the filenames are the same for 2015 and 2017. They can't coexist. The packages need to either have the same name (as they do in defaults) or you need a mutex scheme. The nature of compatibility is such that the name is the best approach.

**Comment by jakirkham** [comment]
Should we update that feedstock then?

**Pull request closed without merging**